app/build.gradle

```diff
@@ -72,6 +72,9 @@ dependencies {
     compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup:otto:1.3.8'
+    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
```

Comment:
yes I am able to use visual effect of highlighting the URL's corresponding
to theme itself.



Sent with MailTrack
<https://mailtrack.io/install?source=signature&lang=en&referral=vishwajeets912@gmail.com&idSignature=22>

*Vishwajeet Srivastava*
B.Tech, Pre-Final (Computer Science and Engineering)
The LNM Institute of Information Technology
Mob:+91-9461787316

On Mon, Jan 25, 2016 at 12:16 AM, Manan Wason <notifications@github.com>
wrote:

> In app/build.gradle
> <https://github.com/fossasia/open-event-android/pull/170#discussion_r50639905>
> :
>
> > @@ -72,6 +72,9 @@ dependencies {
> >      compile "com.android.support:design:${SUPPORT_LIB_VERSION}"
> >      compile 'com.squareup.picasso:picasso:2.5.2'
> >      compile 'com.squareup:otto:1.3.8'
> > +    compile('com.github.afollestad.material-dialogs:core:0.8.5.3@aar') {
>
> No I mean are you using any such functionality?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/170/files#r50639905>.
>